lib/manager/composer/artifacts.ts

```diff
@@ -72,14 +73,14 @@ function getAuthJson(): string | null {
     };
   }
 
-  const gitlabCredentials = hostRules.find({
+  const [gitlabCredentials] = hostRules.findAll({
```

Comment:
Why only the first is used?

Reply:
I changed it, so it will add all gitlab credentials (In case of multiple host rules for gitlab platform) to  `authJson`

Reply:
Nope, it doesn't add all, you only use the first result. you would need to use the all results and iterate over `findAll` result.